client/state/posts/selectors.js

```diff
@@ -287,19 +287,22 @@ export function isRequestingSitePost( state, siteId, postId ) {
  * @param  {Number} postId Post ID
  * @return {Object}        Post object with revisions
  */
-export function getEditedPost( state, siteId, postId ) {
-	const post = getSitePost( state, siteId, postId );
-	const edits = getPostEdits( state, siteId, postId );
-	if ( ! edits ) {
-		return post;
-	}
+export const getEditedPost = createSelector(
+	( state, siteId, postId ) => {
+		const post = getSitePost( state, siteId, postId );
+		const edits = getPostEdits( state, siteId, postId );
+		if ( ! edits ) {
+			return post;
+		}
 
-	if ( ! post ) {
-		return edits;
-	}
+		if ( ! post ) {
+			return edits;
+		}
```

Comment:
I think it's ok, if we have no post and edits, we are just returning a falsy value, which is fine IMO.
I think it's a general practice to return a falsy value from selectors when the data is not loaded yet. That's why we check the existence of the post before checking if the content is empty.